tests/dummy/DummyTypes.elm

```diff
@@ -61,6 +61,11 @@ makeDummyModel sessions =
     , scheduleSubmissionsIndividually = False
     , showMobileView = False
     , displayedColumn = Nothing
+    , filesToSave = [ (FileToSave 0 "" "" "") ]
```

Comment:
[extensible records](https://medium.com/@ckoster22/advanced-types-in-elm-extensible-records-67e9d804030d) can prevent this sort of upkeep.

Reply:
awesome good to know for future

Reply:
Great read @roryc89 hope you don't mind me adding it to: https://github.com/dwyl/learn-elm/issues/102 😉

Reply:
@nelsonic of course! Share away! :)